packages/jellyfish-testing/src/index.ts

```diff
@@ -81,3 +84,50 @@ class TestingJsonRpcClient extends JsonRpcClient {
     })
   }
 }
+
+export class TestingGroup {
+  private constructor (
+    public readonly group: ContainerGroup,
+    public readonly testings: Testing[]
+  ) {
+  }
+
+  static create (n = 1): TestingGroup {
```

Comment:
@canonbrother How about if we address https://github.com/DeFiCh/jellyfish/issues/637 here?
we should be able to pass the containers from outside also as some tests may require it.
maybe we can pass an optional array of containers with `TestingGroup::create(n)`

Reply:
leave the `Testing` class unchanged, as it would require existing test refactoring.

Reply:
https://github.com/DeFiCh/jellyfish/pull/621/files#r703286221

Reply:
yea updated the changes